### PR TITLE
AA-Event-Detail-Page

### PR DESCRIPTION
**Pull request by AntonovAtanas**
Create an event detail page and render data from a mock event.

Added Google Maps with marker which shows a modal when clicked. Used API Key with no restrictions.

Added new dependancies:

- @capacitor/google-maps
- @capacitor/geolocation
- ios
- android

TODO:

- Change API Keys
- Get real event data from service

### CODE REVIEW

File: src/app/events/event-detail/event-detail.page.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        </ion-grid>
+      </div>
+      <div class="category-city-wrapper">
+        <span class="badge">{{mockEvent.category}}</span>
+        <span class="badge">{{mockEvent.contacts.city}}</span>
+      </div>
+      <p>{{mockEvent.longDescription}}</p>
+      <div class="contacts">
+        <h5>Контакти</h5>
+        <p>{{mockEvent.contacts.phone}}<ng-container *ngIf="mockEvent.contacts.email">,
```
Comment (Getsov) [inline src/app/events/event-detail/event-detail.page.html]: Phone and email to be clickable with tel: and mailto:

File: src/app/events/event-detail/event-detail.page.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      phone: '0888888888',
+      email: 'kondofrey@abv.bg',
+      coordinates: { lat: 42.448154, long: 22.963561 },
+    },
+    category: 'Драг',
+    creator: 'Drag Racing Bulgaria',
+    id: '01',
+    isDeleted: false,
+    likedCount: 2,
+    visitorPrice: 15,
```
Comment (Getsov) [inline src/app/events/event-detail/event-detail.page.ts:43]: are the price properties (this and the below one) in the model?

**Review by Getsov — COMMENTED** [review]
2 малки коментара, много добре, поздравления, само гледай да са по-малки PR-ите

File: src/app/events/event-detail/event-detail.page.ts (hunk 2 above)
Comment (AntonovAtanas) [reply src/app/events/event-detail/event-detail.page.ts:43]: Currently they are not in the BE Event model, but we need to have them.

**Review by Getsov — APPROVED** [review]
Very good job!

**Pull request merged**